barretenberg/cpp/src/barretenberg/vm/avm/tests/slice.test.cpp

```diff
@@ -162,9 +164,7 @@ TEST_F(AvmSliceTests, copyToHighestMemOffset)
 
 TEST_F(AvmSliceTests, twoCallsNoOverlap)
 {
-    calldata = { 2, 3, 4, 5, 6 };
```

Comment:
nitpick: If you inline all the local variables calldata, then we could live with the member calldata instead of calldata_